aviary/interface/reports.py

```diff
@@ -38,6 +39,13 @@ def register_custom_reports():
                     method='run_driver',
                     pre_or_post='post')
 
+    register_report(name='timeseries_csv',
+                    func=timeseries_csv,
+                    desc='Generates an output .csv file for variables in the timeseries of the trajectory',
+                    class_name='Problem',
```

Comment:
We should be able to change the class_name from "Problem" to "Aviary_Problem" now because of this bug fix in openmdao: https://github.com/OpenMDAO/OpenMDAO/pull/3125

Reply:
Thanks Ken, changed now!